oaknational/Oak-Web-Application
# Feat/pupil 449/explanatory text pt 2

### User Story

**As a** pupil completing a lesson,
**I want** to view explanatory text whilst taking quizzes,
**So that** I can gain useful context about the questions I am about to complete

### Implementation details

Example legacy lesson:
https://www.thenational.academy/pupils/lessons/sonnet-18-the-shakespearean-sonnet-68u62d

https://www.figma.com/file/G7z5ufo2T0CHk14CkadeuB/Pupils-high-fidelity-designs?node-id=3034%3A171415&mode=dev

### To test
- Go to https://deploy-preview-2274--oak-web-application.netlify.thenational.academy/pupils/lessons/sonnet-18-the-shakespearean-sonnet-68u62d
- Take the exit quiz
- See that the explanatory text is rendered and the lesson counts remain correct

### Acceptance criteria

- [x]  rendered during quiz
- [x]  not included in the question counts during quiz
- [x]  ‘completing’ the explanatory text question does not increase the user’s score by 1
- [x]  not included in the question counts for overview and review sections
